### PR TITLE
Remove dependency on parity_wasm from chisel CLI

### PR DESCRIPTION
**Pull request by axic**
Follow up of #154. Also rolls back #150.

### CODE REVIEW

File: chisel/src/main.rs [hunk 1]
```diff
@@ -327,12 +325,13 @@ fn chisel_execute(context: &ChiselContext) -> Result<bool, &'static str> {
 
             // If the module was mutated, serialize to file.
             if original != module {
+                let serialized = module.to_bytes().unwrap();
                 if let Some(path) = context.outfile() {
                     chisel_debug!(1, "Writing to file: {}", path);
-                    serialize_to_file(path, module).unwrap();
+                    write(path, serialized).unwrap();
```
Comment (jakelang) [inline chisel/src/main.rs]: Can we use `expect` with a little description that the write operation failed for some reason?

File: chisel/src/main.rs [hunk 2]
```diff
@@ -327,12 +325,13 @@ fn chisel_execute(context: &ChiselContext) -> Result<bool, &'static str> {
 
             // If the module was mutated, serialize to file.
             if original != module {
+                let serialized = module.to_bytes().unwrap();
```
Comment (jakelang) [inline chisel/src/main.rs]: Same here but for serialization failure.

File: libchisel/src/lib.rs [hunk 3]
```diff
@@ -3,7 +3,7 @@ extern crate binaryen;
 extern crate parity_wasm;
 extern crate rustc_hex;
 
-use parity_wasm::elements::Module;
+pub use parity_wasm::elements::Module;
```
Comment (jakelang) [inline libchisel/src/lib.rs]: Not needed yet, but perhaps we could benefit from a prelude of sorts containing re-exports and the like.

File: chisel/src/main.rs (hunk 1 above)
Comment (axic) [reply chisel/src/main.rs]: We could but your previous wasn't doing that either and I expected the big CLI PR will deal with it so wanted to minise the change here.

File: libchisel/src/lib.rs (hunk 3 above)
Comment (axic) [inline libchisel/src/lib.rs]: Though since this is now exported, we could remove `module_from_bytes`.

**Pull request merged**